tests/scripts/test_reporting.py

```diff
@@ -220,3 +222,33 @@ def _load_json(a_file):
             # schemas directory contain a symlink to report json-schemas
             schema_data = _load_json(os.path.join(reports_dir, 'schemas', schema))
             jsonschema.validate(report_data, schema_data)
+
+
+def test_report_backwards_compatibility():
+    msg1 = {u'groups': [u'selinux', u'security'],
+            u'title': u'SElinux relabeling has been scheduled',
+            'timeStamp': u'2021-02-10T13:13:40.576437Z',
+            'hostname': u'ivasilev-tagsflagsgroups',
+            'actor': u'check_se_linux',
+            u'summary': u'SElinux relabeling has been scheduled as the status was permissive/enforcing.',
+            u'audience': u'sysadmin',
+            u'key': u'c12a05a22be0b5bc0af3f1119898ea6d8639d9c4',
+            'id': '29279fc2e3100cfdf8ac30f48b293185192672d5a5ae9e0cfbc76cbfc919d807',
+            u'severity': u'info'}
+    report = [msg1]
+    # make sure normal mode works
+    for report_format in ['.json', '.txt']:
+        reportfile = tempfile.NamedTemporaryFile(suffix=report_format)
+        generate_report_file(report, 'leapp-run-id', reportfile.name, '1.1.00')
```

Comment:
I'm a bit confused about the `1.1.00`... why is it there?

also feel free to squash my commit, I don't mind :)

Reply:
Looks like I need to call it a day and finally clean the keyboard. Thanks, will fix.
As for the squashing - I'd rather keep 2 separate commits :)